hack/jenkins/upload-to-gcs.sh

```diff
@@ -64,19 +64,36 @@ readonly results_url=${gcs_build_path//"gs:/"/"https://console.cloud.google.com/
 readonly timestamp=$(date +%s)
 
 #########################################################################
+# $0 is called from different contexts so figure out where kubernetes is.
+# Sets non-exported global kubernetes_base_path and defaults to "."
+function set_kubernetes_base_path () {
+  for kubernetes_base_path in kubernetes .; do
```

Comment:
if we want to be super thorough, `go/src/k8s.io/kubernetes` also appears in some places (like node e2e)